src/site/content/en/lighthouse-accessibility/document-title/index.md

```diff
@@ -1,71 +1,85 @@
 ---
 layout: post
-title: Ensure that each HTML document contains a title
+title: Document doesn't have a <title> element
 description: |
-  Learn about document-title audit.
+  Learn about the "Document doesn't have a <title> element"
+  Lighthouse audit.
```

Comment:
Not gonna block it here, but for future reference, we should make these more descriptive.

Reply:
Do you have a suggested pattern?